app/components/create-test/view.jsx

```diff
@@ -274,7 +278,24 @@ export default React.createClass({
 		}
 	},
 	renderQuestions() {
-		const questions = this.state.questions;
+		let questions = this.state.questions;
+		const testId = this.props.params.testId;
+		let submittedAnswers = [];
+		const testResults = this.state.user.test_results || {};
+
+		if (testResults[testId] !== undefined) {
+			submittedAnswers = testResults[testId].answers;
+			questions = questions.map((question) => {
+				let userAnswer;
+				submittedAnswers.forEach((answer) => {
+					if (answer.id === question._id) {
+						userAnswer = answer.actual;
+					}
+				});
+				return {...question, userAnswer };
+			});
```

Comment:
I wonder if something like this might work 
```js
let userAnswer;
userAnswer = submittedAnswers.filter((answer) => answer.id === question._id);
return {...question, userAnswer: userAnswer.actual };
```